fuzz/fuzz_targets/cranelift-fuzzgen.rs

```diff
@@ -325,12 +336,42 @@ fuzz_target!(|testcase: TestCase| {
             RunResult::Error(_) => panic!("interpreter failed: {:?}", int_res),
         }
 
-        let host_res = run_in_host(&trampoline, args);
-        match host_res {
+        let res = run(args);
+
+        match res {
             RunResult::Success(_) => {}
-            _ => panic!("host failed: {:?}", host_res),
+            _ => panic!("second run failed: {:?}", res),
         }
 
-        assert_eq!(int_res, host_res);
+        assert_eq!(int_res, res);
+    }
+}
+
+fuzz_target!(|testcase: TestCase| {
+    // This is the default, but we should ensure that it wasn't accidentally turned off anywhere.
+    assert!(testcase.isa.flags().enable_verifier());
+
+    // Periodically print statistics
+    let valid_inputs = STATISTICS.valid_inputs.fetch_add(1, Ordering::SeqCst);
+    if valid_inputs != 0 && valid_inputs % 10000 == 0 {
+        STATISTICS.print(valid_inputs);
+    }
+
+    if testcase.should_optimize {
+        let opt_testcase = testcase.to_optimized();
+
+        run_test_inputs(&testcase, |args| {
+            let mut interpreter = build_interpreter(&opt_testcase);
+            run_in_interpreter(&mut interpreter, &args)
```

Comment:
I don't think `args` needs to be borrowed here, right? It's already a `&[DataValue]` which is what `run_in_interpreter` expects.
```suggestion
            run_in_interpreter(&mut interpreter, args)
```